### ISSUE TITLE
Condition strain requires strain-high-pass

### ISSUE THREAD

**Issue by cdcapano** [gwastro/pycbc#1947]
When running `pycbc_condition_strain` with `fake-strain` you apparently need to specify `strain-high-pass`. This is because of [this line](https://github.com/ligo-cbc/pycbc/blob/master/pycbc/strain.py#L374) in strain.py. This isn't intuitive. It seems odd that you need to use a high-pass filter for fake strain, especially since a `low-frequency-cutoff` option exists. But I'm not sure what the best solution is, since I don't know if the low-frequency-cutoff option is always provided. At the very least, a more informative error than the following should be provided:
```
  File "/home/cdcapano/.virtualenvs/pycbc_master/lib/python2.7/site-packages/PyCBC-a3d10f-py2.7.egg/pycbc/strain.py", line 374, in from_cli
    low_frequency_cutoff=opt.strain_high_pass)
  File "/home/cdcapano/.virtualenvs/pycbc_master/lib/python2.7/site-packages/PyCBC-a3d10f-py2.7.egg/pycbc/noise/reproduceable.py", line 149, in colored_noise
    kmin = int(low_frequency_cutoff / psd.delta_f)
TypeError: unsupported operand type(s) for /: 'NoneType' and 'float'
```

**Comment by titodalcanton** [comment]
Also, there seems to be an inconsistency between the starting frequency used by `colored_noise()` and the one used for PSD generation [some lines above](https://github.com/ligo-cbc/pycbc/blob/master/pycbc/strain.py#L357). The former takes the frequency from `--strain-high-pass` while the latter uses `--low-frequency-cutoff`. Maybe there should be a new option `--fake-strain-min-freq` or so and both operations would use that?

**Comment by micamu** [comment]
Without knowing about this issue, I just decided to make some changes to pycbc_condition_strain after trying to teach a new user how to run it and getting cryptic error messages. That is the origin of PR #2140, but it seems like you would want something different (i.e. the colored_noise low frequency to be the same as the PSD generation low frequency). Let me know in the corresponding PR if you want me to change it to be the same frequency for both (in which case I would just suggest to just leave it as low-frequency-cutoff)

**Comment by micamu** [comment]
This issue was addressed by PR#2140
